fractal_server/utils.py

```diff
@@ -74,6 +100,8 @@ async def execute_command(*, cwd: Path, command: str) -> str:
     command_split = shlex_split(command)
     cmd, *args = command_split
 
+    logger = set_logger(logger_name=logger_name)
```

Comment:
Should you also handle `logger_name=None`?

Reply:
when None, `getLogger` returns the default logger.